### PR TITLE
feat: mentions

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

File: src/element/Textarea.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import Nip05, { useIsVerified } from "./Nip05";
+import "@webscopeio/react-textarea-autocomplete/style.css";
+import "./Textarea.css";
+import Nostrich from "../nostrich.jpg";
+
+function searchUsers(query, users) {
+  const q = query.toLowerCase()
+  return Object.values(users).filter(({ name, display_name, about }) => {
+    return name.toLowerCase().includes(q) || display_name?.includes(q) || about?.includes(q)
```
Comment (v0l) [inline src/element/Textarea.js:13]: nip5 includes

File: src/element/Textarea.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Component } from "react";
```
Comment (v0l) [inline src/element/Textarea.js:1]: This should be `tsx` now, can you convert it to TS?

File: src/feed/EventPublisher.js [hunk 3]
```diff
@@ -28,6 +30,25 @@ export default function useEventPublisher() {
         return ev;
     }
 
+
+    function processMentions(ev, msg) {
+        const replaceHexKey = (match) => {
```
Comment (v0l) [inline src/feed/EventPublisher.js:35]: Trying to remove hex keys i think, only `npub` / `note` keys should be extracted

Comment (verbiricha) [reply src/feed/EventPublisher.js]: Leaving note references out for now, this PR only adds pubkey mentions with bech32 notation.

**Comment by verbiricha** [comment]
Should be much better after https://github.com/v0l/snort/issues/47 but this is a start.

**Review by v0l — APPROVED** [review]

**Pull request merged**